contrib/google-cloud-cpp-cmake/CMakeLists.txt

```diff
@@ -11,13 +11,28 @@ if(NOT ENABLE_GOOGLE_CLOUD_CPP)
     return()
 endif()
 
+if(CLICKHOUSE_CLOUD)
+    set(ENABLE_GOOGLE_CLOUD_CPP_KMS_DEFAULT ON)
+endif()
+
+option(ENABLE_GOOGLE_CLOUD_CPP_KMS "Enable Google Cloud Cpp KMS" ${ENABLE_GOOGLE_CLOUD_CPP_KMS_DEFAULT})
+
 if(NOT ENABLE_GRPC)
-    message (${RECONFIGURE_MESSAGE_LEVEL} "Can't use Google Cloud Cpp without gRPC")
+    message(STATUS "Can't use Google Cloud Cpp without gRPC")
```

Comment:
Guess this should be a FATAL_ERROR. Silently not building gcp despite the user requested it is worse.

Reply:
I modified it so that it doesn't throw a failure when configuring the flavors: riscv64, s390x, loongarch64, ppc64le.
What I'm going to do is to explicitly enable `GOOGLE_CLOUD_CPP` only for those flavors we know it works in: Linux, macOS and FreeBSD, ok?

Reply:
Yepp, tnx